distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -926,12 +935,23 @@ public String getOption() {
               throw new RuntimeException(UNABLE_TO_DETERMINE_EXAM_DIR_ERROR, e);
             }
           };
+      System.err.println("HomeAwareVmOption");
```

Comment:
✏️ I would prefer using the logger on line 111 for all these findings.

Reply:
Removed